src/logic/wallets/ethTransactions.ts

```diff
@@ -1,7 +1,9 @@
 import axios from 'axios'
 import { BigNumber } from 'bignumber.js'
+import { EthAdapterTransaction } from '@gnosis.pm/safe-core-sdk/dist/src/ethereumLibs/EthAdapter'
```

Comment:
It looks like I forgot to export `EthAdapterTransaction`. I will update this import once it is exported properly in the next version. https://github.com/gnosis/safe-core-sdk/issues/66